### PR TITLE
add roles model and table with name and slug fields to db

### PR DESCRIPTION
**Pull request by VickTor61**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -24,6 +24,22 @@
...
   end
 
+  create_table "roles", force: :cascade do |t|
+    t.string "name"
+    t.string "slug"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "user_roles", force: :cascade do |t|
```
Comment (EnomaDebby) [inline db/schema.rb]: Roll back this migration and push. I don’t want to see this in this PR

File: app/models/role.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+class Role < ApplicationRecord
```
Comment (EnomaDebby) [inline app/models/role.rb:1]: Add presence validation for name and slug

File: db/schema.rb (hunk 1 above)
Comment (VickTor61) [reply db/schema.rb]: ok

File: app/models/role.rb (hunk 2 above)
Comment (VickTor61) [reply app/models/role.rb:1]: done

**Review by EnomaDebby — APPROVED** [review]

**Pull request merged**